api/apps/api/src/modules/projects/project-cost-surface.controller.ts

```diff
@@ -263,7 +263,6 @@ export class ProjectCostSurfaceController {
 
   @ImplementsAcl()
   @UseGuards(JwtAuthGuard)
-  @TilesOpenApi()
```

Comment:
@yulia-bel apologies, I had completely missed that this decorator was in place (and I had actually completely forgotten about it even existing). This was correct, we had all the params documented via this decorator. I just meant that we would not necessarily need OpenAPI docs for the _geoprocessing_ endpoint, but here all was correct - sorry!

The only glitch here is that `@TilesOpenApi()` includes the `@ApiQuery` decorator for `bbox`, which we may not want here. If we wanted to _optionally exclude_ the bbox param (so that we don't have to touch any of the existing uses of the `@TilesOpenApi()` decorator), we could change the decorator with something like the patch below

```patch
diff --git a/api/libs/tiles/src/tiles-api-doc.ts b/api/libs/tiles/src/tiles-api-doc.ts
index 7beb84368..965d0ca5f 100644
--- a/api/libs/tiles/src/tiles-api-doc.ts
+++ b/api/libs/tiles/src/tiles-api-doc.ts
@@ -7,7 +7,16 @@ import {
   ApiUnauthorizedResponse,
 } from '@nestjs/swagger';

-export const TilesOpenApi = () =>
+export const TilesOpenApi = (excludeBbox: boolean) => {
+  const bboxDecorator = excludeBbox ? [] : [
+      ApiQuery({
+        name: 'bbox',
+        description: 'Bounding box of the project [xMin, xMax, yMin, yMax]',
+        type: [Number],
+        required: false,
+        example: [-1, 40, 1, 42],
+      }),
+  ];
   applyDecorators(
     ...[
       ApiParam({
@@ -35,13 +44,6 @@ export const TilesOpenApi = () =>
         required: false,
         example: 'e5c3b978-908c-49d3-b1e3-89727e9f999c',
       }),
-      ApiQuery({
-        name: 'bbox',
-        description: 'Bounding box of the project [xMin, xMax, yMin, yMax]',
-        type: [Number],
-        required: false,
-        example: [-1, 40, 1, 42],
-      }),
       ApiOkResponse({
         description: 'Binary protobuffer mvt tile',
         type: String,
@@ -49,4 +51,6 @@ export const TilesOpenApi = () =>
       ApiBadRequestResponse(),
       ApiUnauthorizedResponse(),
     ],
-  );
+    ...bboxDecorator
+    );
+  };
```